yarn-project/circuit-types/src/stats/metrics.ts

```diff
@@ -122,6 +122,48 @@ export const Metrics = [
     description: 'Time for the sequencer to execute public function calls for txs in a block.',
     events: ['l2-block-built'],
   },
+  {
+    name: 'setup_kernel_simulation_avg_ms',
+    groupBy: 'block-size',
+    description: 'Average time to simulate a kernel circuit during setup.',
+    events: ['l2-block-built'],
+  },
+  {
+    name: 'setup_avm_simulation_avg_ms',
+    groupBy: 'block-size',
```

Comment:
I think the AVM simulation stats need to be grouped by function name as they depend on the size/complexity of the bytecode being executed.